# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/tsa/metamail/70_mm-2.7-sendmail.patch
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

diff -Nur mm2.7.orig/src/bin/showexternal mm2.7/src/bin/showexternal
--- mm2.7.orig/src/bin/showexternal	2003-05-14 23:58:27.000000000 +0200
+++ mm2.7/src/bin/showexternal	2003-05-14 23:59:21.000000000 +0200
@@ -219,7 +219,7 @@
 	echo "" >> "$NEWNAME"
 	sed -e 1,/^\$/d < $bodyfile >> "$NEWNAME"
 	echo "" >> "$NEWNAME"
-	/usr/lib/sendmail -t  < "$NEWNAME"
+	/usr/sbin/sendmail -t  < "$NEWNAME"
 	if ($status) then
 	    echo sendmail failed
 	    cd ${METAMAIL_TMPDIR}
